docs/push.md

```diff
@@ -91,10 +91,22 @@ oci_image_index(
     ]
 )
 
-# This is defined in our /examples/push
-stamp_tags(
+write_file(
+    name = "tags_tmpl",
+    out = "tags.txt.tmpl",
+    content = [
+        "BUILD_VERSION",
+    ],
+)
+
+# Use the value of --embed_label under --stamp, otherwise use a deterministic constant
+# value to ensure cache hits for actions that depend on this.
+expand_template(
     name = "stamped",
-    remote_tags = ["""($stamp.BUILD_EMBED_LABEL // "0.0.0")"""],
+    out = "_stamped.tags.txt",
+    template = "tags_tmpl",
```

Comment:
note: I think we should change expand_template to a macro that accept a list of strings here, that would create the write_file target for you

Reply:
https://github.com/aspect-build/bazel-lib/pull/533

Not sure if I want to land/release that first, then come back to this PR and force users onto newest bazel-lib. Probably... WDYT?

Reply:
given that this is just docs, I'd say we shouldn't wait for it.